WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -124,18 +130,18 @@ private extension ItemListView {
     func listView(_ items: [POSItem]) -> some View {
         ScrollView {
             VStack {
-                if dynamicTypeSize.isAccessibilitySize, viewModel.shouldShowHeaderBanner {
+                if dynamicTypeSize.isAccessibilitySize, shouldShowHeaderBanner {
                     bannerCardView
                 }
                 ForEach(items, id: \.productID) { item in
                     Button(action: {
-                        viewModel.select(item)
+                        posModel.addToCart(item)
```

Comment:
It's nice to remove this extra layer of indirection, and just add items to cart directly from the view.